### PR TITLE
Minor code fixes

### PR DESCRIPTION
**Pull request by lccezinha**
### WTF?

This PR main intent is to:

- Remove the use of `var` to use `let` and `const` instead
- Minor fixes

Also, I fixed a lot of eslint warning (except in the `office.web.js`, I think is best to do it in a separated PR, to make the review easier)

:warning: Before ship it, it's important to validate the whole flow (login/logout, enter/leave rooms).

### CODE REVIEW

File: public/js/office.web.js [hunk 1]
```diff
@@ -23,51 +20,48 @@ $(() => {
...
   function showUserInRoom(user, room) {
+    let userView = $(`#${user.id}`).length;
 
-    var userView = $(`#${user.id}`).length;
     if (userView == 0) {
       userView = $(`<div  id="${user.id}" class="thumbnail user-room"><img user-presence class="rounded-circle" style="margin:2px;display:flex;" user-id="${user.id}" title="${user.name}" width="50px" src="${user.imageUrl}"></div>`);
-     } else {
-       userView = $(`#${user.id}`).detach();
+    } else {
+      userView = $(`#${user.id}`).detach();
```
Comment (alesshh) [inline public/js/office.web.js]:
```suggestion
      userView.detach();
```

File: public/js/office.web.js [hunk 2]
```diff
@@ -23,51 +20,48 @@ $(() => {
...
+    });
   }
 
   function userInMeetDecorator(user,userView){
+    let userMeetClass = "rounded-circle user-not-in-call user-room"
 
-    var userMeetClass = "rounded-circle user-not-in-call user-room"
-
-    if(user.inMeet!=undefined && user.inMeet){
+    if (user.inMeet !== undefined && user.inMeet) {
```
Comment (alesshh) [inline public/js/office.web.js]:
```suggestion
    if (user.inMeet) {
```

Maybe this code can be refactored to simplify conditional classes like:
```
     userview.toggleClass('user-in-call', user.inMeet)
     userview.toggleClass('user-not-call', !user.inMeet)
}
````

`rounded-circle ` and `user-room"` classes should be default at `userView` creation

File: public/js/office.web.js [hunk 3]
```diff
@@ -128,7 +124,7 @@ $(() => {
 
   function isUserInVideoConference(){
     if($("#exampleModalCenter").data('bs.modal')){
```
Comment (alesshh) [inline public/js/office.web.js]: It can be cached to a variable to to avoid dom fetch like:

```
const modal = $("#exampleModalCenter")
if (modal.data('bs.modal') {
    modal...
}

**Comment by lccezinha** [comment]
Hey, @alesshh I made some changes, thanks for your review! Could you please review it again?!

File: public/js/office.web.js [hunk 4]
```diff
@@ -23,83 +20,83 @@ $(() => {
...
 
-  function userInMeetDecorator(user,userView){
-
-    var userMeetClass = "rounded-circle user-not-in-call user-room"
+  function userInMeetDecorator(user, userView){
+    const userMeetDefaultClasses = "rounded-circle user-room";
 
-    if(user.inMeet!=undefined && user.inMeet){
-      userMeetClass = "rounded-circle user-in-call user-room";
+    if (user.inMeet) {
```
Comment (alesshh) [inline public/js/office.web.js]: This check is unnecessary (and introduce a bug), given that toggleClass will handle with classes when user enter or leave a meet.

File: public/js/office.web.js [hunk 5]
```diff
@@ -23,83 +20,83 @@ $(() => {
...
 
-    var userView = $(`#${user.id}`).length;
-    if (userView == 0) {
+    if (!userView.length) {
       userView = $(`<div  id="${user.id}" class="thumbnail user-room"><img user-presence class="rounded-circle" style="margin:2px;display:flex;" user-id="${user.id}" title="${user.name}" width="50px" src="${user.imageUrl}"></div>`);
-     } else {
-       userView = $(`#${user.id}`).detach();
+    } else {
+      userView.detach();
     }
```
Comment (alesshh) [inline public/js/office.web.js]: Maybe you can invert this check, to avoid if with negative condition.

Comment (lccezinha) [reply public/js/office.web.js]: Make sense.

**Comment by lccezinha** [comment]
@alesshh Could you please, check it again? Also, I have an issue with CC, but for me makes more sense to keep the code easy to read instead use a DRY approach, could you check it too, please?

**Comment by lccezinha** [comment]
@juliemar Could you plz check this one?
